src/components/Newsletter/Newsletter.js

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
-import { StyledNewsletter } from './Newsletter.style';
+import { StyledNewsletter, InputHolder } from './Newsletter.style';
+import Chip from '@material-ui/core/Chip';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import TextField from '@material-ui/core/TextField';
+import { each } from 'lodash';
+import US_STATE_DATASET from 'components/MapSelectors/datasets/us_states_dataset_01_02_2020';
 
 class Newsletter extends React.Component {
   constructor() {
     super();
     this.form = null;
     this.emailInput = null;
+    this.alertsSelectionArray = [];
```

Comment:
Can we initialize this with the current state / county page that you're on?

Reply:
done